README.md

```diff
@@ -1,48 +1,521 @@
+please follow [Github Flow](https://githubflow.github.io/).
 
-## How to use Make?
+```shell script
+# get the code:
+git clone git@github.com:CodeForPoznan/pah-fm.git
 
-We use Make to make to manage running our project.
-Run this command in terminal to see what Make commands are available:
-```bash
-make
+
+# if you intend to send a patch then clone your fork:
+git clone git@github.com:<your github username>/pah-fm.git
+
+# i.e:
+git clone git@github.com:arturtamborski/pah-fm.git
 ```
 
-## How to debug backend?
-Run in terminal:
-```bash
-make debug-backend
+
+2. Go to project directory:
+
+```shell script
+cd pah-fm
+```
+
+
+3. Start the project  
+
+```shell script
+# This command will download Docker images set up local stack.
+make start
+```
+
+
+4. Generate some random data for testing
+
+```shell script
+make populate-database
+```
+
+
+5. Open up [http://localhost:8080/](http://localhost:8080/) and use the app!
+There's also admin page [http://localhost:8080/admin/](http://localhost:8080/admin/)
+
+
+At this point you'll be able to log in as driver or passenger,
+submit a drive, optionally verify it, view your previous drives,
+change language, manage drives from admin panel or export them to
+common data formats such as CSV.
+
+
+<br>
+
+### How to Debug the Project or Start Working on Pull Request?
+
+First, get familiar with project's handy toolset - `make`!  
+We wrapped many common actions that you'd want to execute 
+as a developer in a `Makefile`.
+
+You can view them all by running `make help`.
+```shell script
+pah-fm$ make
+
+Usage:
+  make <target>
+
+Targets:
+               help   Display this help
+              start   Start all containers in background
+               stop   Stop all containers
+              build   Build backend & frontend containers
+            adminer   run database adminer
+             remove   Stop and remove backend & frontend containers
+            rebuild   Rebuid application
+  rebuild-docker-images   Rebuid backend & frontend containers
+               logs   Attach to logs
+               lint   Run linters
+               test   Run tests
+             manage   Use manage.py, i.e make manage CMD=collectstatic
+      build-backend   Build backend container
+     build-frontend   Build frontend container
+     remove-backend   Stop and remove backend container
+    remove-frontend   Stop and remove frontend container
+       lint-backend   Run linters on backend container
+      lint-frontend   Run linters on frontend container
+      test-frontend   Run tests on frontend container
+       test-backend   Run tests on backend container
+       bash-backend   Enter backend container
+      bash-frontend   Enter frontend container
+      debug-backend   Debug backend container (Django)
+  populate-database   Populate database with factory based data
+           checkout   Checkout to existing branch and start clean app, i.e. make checkout BRANCH=develop
```

Comment:
```
... BRANCH=develop
```
The description for this options should be updated as well. The script itself looks fine.